pkgs/development/python-modules/weasyprint/default.nix

```diff
@@ -45,7 +46,13 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ cairosvg pyphen cffi cssselect lxml html5lib tinycss pygobject2 ];
```

Comment:
Could you please [remove](https://github.com/Kozea/WeasyPrint/blob/16725609095b6839e2cf2313e84e1ff052b41a49/pyproject.toml#L13) `pygobject2` from dependencies to allow build with `python39`?

Reply:
I noticed that, too, but I only found this package today when I went to find a command-line util to take a simple HTML file and convert it to PDF. It seems this package can do all sorts of things that are more complex than that so I'm a little wary of just removing it in this PR.

That said it builds fine without `pygobject2` and even at release `v45` where the package was added to nixpgs (https://github.com/NixOS/nixpkgs/pull/57377) I can't find evidence of the code making use of `pygobject2`.

So how confident are you that `pygobject2` is not needed?

Reply:
I think it's because long ago they depended on Cairo, but [they got rid of that dependency](https://github.com/Kozea/WeasyPrint/issues/841#issuecomment-761569954). So... probably safe?

Reply:
I think the absence of `pygobject2` in their [pyproject.toml](https://github.com/Kozea/WeasyPrint/blob/16725609095b6839e2cf2313e84e1ff052b41a49/pyproject.toml#L13) speaks for itself and since tests are passing it should be safe to remove.

Reply:
Done.